### ISSUE TITLE
Yii Debug Toolbar conflict

### ISSUE THREAD

**Issue by dletsche** [ethercreative/seo#121]
### Description

Enabling this plugin stops the Yii Debug Toolbar from displaying on the frontend (it continues to work in the Control Panel).

### Steps to reproduce

1. Check "Show the debug toolbar on the front end" under your account preferences (/admin/myaccount#prefs).
2. Visit the homepage and note the debug toolbar is present.
3. Enable this plugin.
4. Visit the homepage again and note the debug toolbar is gone. And note a 404 message in the browser's developer console. 

### Additional info

- Craft version: Craft Pro 3.0.16.1
- SEO version: 3.3.0
- PHP version: 7.2.1
- Database driver & version: MySQL 5.6.38
- Other Plugins:

**Comment by Tam** [comment]
I can't re-create this. What other plugins are you using, and would you be able to send me the 404 log for the debug bar?

**Comment by dletsche** [comment]
Very sorry for the delayed reply! I got back into the project today and remembered this. I somehow missed/ignored the notification that you replied.

Anyway, the issue appears to be resolved.

If it's worth knowing, I ran all Craft CMS updates today and the issue was still there. I then ran all SEO updates, and the issue was resolved. So, perhaps you fixed it in a recent update? Or perhaps some other strange scenario with my setup/plugins cleared up the issue.

Thanks for the plugin!
